lib/wordmove/movefile.rb

```diff
@@ -75,18 +73,40 @@ def fetch(verbose = true) # rubocop:disable Style/OptionalBooleanParameter
       end
 
       found = entries.first
+
       logger.task("Using Movefile: #{found}") if verbose == true
-      YAML.safe_load(ERB.new(File.read(found)).result, [], [], true).deep_symbolize_keys!
+      load_dotenv(verbose)
+
+      options = YAML.safe_load(ERB.new(File.read(found)).result, [], [], true).deep_symbolize_keys!
+
+      merge_local_options_from_wpcli(options)
+    end
+
+    def merge_local_options_from_wpcli(options)
+      config_path = options.dig(:local, :wordpress_path)
+
+      options.merge(
+        local: {
+          database: {
+            password: Wordmove::WpcliHelpers.get_config('DB_PASSWORD', config_path: config_path),
+            host: Wordmove::WpcliHelpers.get_config('DB_HOST', config_path: config_path),
+            name: Wordmove::WpcliHelpers.get_config('DB_NAME', config_path: config_path),
+            user: Wordmove::WpcliHelpers.get_config('DB_USER', config_path: config_path)
+          },
+          vhost: Wordmove::WpcliHelpers.get_option('siteurl', config_path: config_path),
```

Comment:
I think It should be `home` not `siteurl` which in a "Bedrock" like structure will return something like `https://www.example.com/wp`

Reply:
Woah. This is a terrific catch!

Going to fix it in #624 since this is already merged :)